### PR TITLE
add config for mainnet version

### PR DESCRIPTION
**Pull request by ghost-not-in-the-shell**
Explain your changes:
This PR adds a new compile config that's almost the same as the `mainnet.mlh` but the protocol version refers to 1. This is mainly used to build `berkeley_migration` app and `replayer` to do the migration

Explain how you tested your changes:
*

Checklist:

- [ ] Dependency versions are unchanged
  - Notify Velocity team if dependencies must change in CI
- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

* Closes #0000

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Review by stevenplatt — APPROVED** [review]

**Review by georgeee — APPROVED** [review]

File: src/config/version1.mlh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+[%%define ledger_depth 35]
+[%%import "/src/config/curve/medium.mlh"]
```
Comment (deepthiskumar) [inline src/config/version1.mlh:2]: Why is this needed if we are updating the txn version using sql? 
If it is required then is this config to be used only for generating the migration app for migrating mainnet DB and not for the daemon builds? If so, I'd suggest naming it appropriately- maybe `berkeley_archive_migration` and creating a separate build job for the app.  Otherwise this could cause some confusion

**Review by deepthiskumar — CHANGES_REQUESTED** [review]

File: src/config/version1.mlh (hunk 1 above)
Comment (ghost-not-in-the-shell) [reply src/config/version1.mlh:2]: This is only needed to for the replayer. I would rename it as you suggested.
